neow3j-test/build.gradle

```
@@ -0,0 +1,18 @@
+description 'test tools for neow3j'
```

```suggestion
description 'test framework for neow3j'
```

or do you prefer to leave as "tools"? no strong feelings here.